src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
```

Comment:
The 'body' tag should not have 'margin: 0 auto'. This rule is used to center block elements horizontally, but since 'body' is not a block element that you would typically center in this way, it's unnecessary here.